code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -697,14 +697,18 @@ var/list/ai_list = list()
 			var/list/icon_list = list(
 				"default" = 'icons/mob/AI.dmi',
 				"floating face" = 'icons/mob/AI.dmi',
-				"xeno queen" = 'icons/mob/AI.dmi',
+				"xeno queen" = 'icons/mob/alien.dmi',
 				"horror" = 'icons/mob/AI.dmi'
 				)
 
 			input = input("Please select a hologram:") as null|anything in icon_list
 			if(input)
 				qdel(holo_icon)
-				holo_icon = getHologramIcon(icon(icon_list[input], input))
+				switch(input)
+					if("xeno queen")
+						holo_icon = getHologramIcon(icon(icon_list[input],"alienq"))

```

Comment:
This does not automatically turn the xeno queen blue; you need to replace the existing icon with an updated blue version